<ui version="4.0" >
 <class>NMGSyncWidgetBase</class>
 <widget class="QWidget" name="NMGSyncWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="Label1_2" >
     <property name="minimumSize" >
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Select a Host:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NMGHostInformationComboWidget" native="1" name="sourceBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Label1_2_2" >
     <property name="minimumSize" >
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Select an INTERVAL between mesures and press START to begin collecting data and STOP to get the result file:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="spacer9_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="sleepbox" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>180</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>seconds</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer9" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>397</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer2_2_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="resultat7" >
       <property name="minimumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer2_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="resultat3" >
       <property name="minimumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="resultat2" >
       <property name="minimumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer16" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="resultat" >
       <property name="minimumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer16_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="resultat4" >
       <property name="minimumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer16_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="resultat5" >
       <property name="minimumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>410</width>
         <height>35</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer2_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer name="spacer4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>360</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string>Stop</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string>St&amp;art</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>NMGHostInformationComboWidget</class>
   <extends>QWidget</extends>
   <header>nmghostinformationcombowidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startButton</tabstop>
 </tabstops>
 <includes>
  <include location="local" >nmghostinformationcombowidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
